3.10/http/document-address-and-etag.md

```diff
@@ -163,3 +163,87 @@ unchanged, a *HTTP 412* (precondition failed) error is returned.
 If you want to query, replace, update or delete a document, then you
 can use the *If-Match* header. If the document has changed, then the
 operation is aborted and an *HTTP 412* error is returned.
+
+Dirty Reads
+-----------
+
+Usually, in an ArangoDB cluster, all reads and writes are performed via
+the shard leaders. Shard replicas replicate all operations, but are
+only on hot standby to take over in case of a failure. This is to ensure
+consistency of reads and writes and allows to give a certain amount of
+transactional guarantees.
+
+Sometimes, high throughput is more important than consistency and
+transactional guarantees. Therefore a number of read-only operations
+can be configured to allow for so-called "dirty reads", or "read from
+followers". In this case, coordinators are allowed to read not only from
+shard leaders but also from shard replicas. This has a positive effect,
+because the reads can scale out to all dbservers which have copies of
+the data. Therefore, read throughput is higher. Note however, that you
+still have to go through your coordinators. So to reap the benefits you
+have to have enough coordinators, load balance your client requests
+across all of them, and then allow for dirty reads.
+
+Obviously, there is no free lunch. Therefore, dirty reads have
+disadvantages, too. Namely, they are, well, "dirty". A number of strange
+artifacts can happen when performing dirty reads:
+
+  1. It is possible to see an old, **obsolete revision** of a document. More
+     exactly, it is possible that some document is already updated on the
+     leader, but the update has not yet been replicated to the follower
+     from which we are reading.
+
+  2. It is also possible to see an update to a document which
+     **has already happened on a replica**, but is not yet officially
+     committed on its leader.
+
+Both of these artifacts will be relatively rare, since usually, ArangoDB
+replicates changes from the leader to the followers in a nearly
+synchronous fashion. However, in particular when performing larger
+transactions, intermediate commits may happen on followers, and then
+dirty artifacts are more likely. This is also the case if failures of
+instances in a cluster happen.
+
+In general, when no writes are happening, allowing dirty reads is safe.
+
+Currently, the following APIs support dirty reads:
+
+ - single document reads (`GET /_api/document`)
+ - batch document reads (`PUT /_api/document?onlyRead=true`)
```

Comment:
It's `onlyget` according to the docs:
https://www.arangodb.com/docs/stable/http/document-working-with-documents.html#read-multiple-documents

Reply:
This is an oversight, thanks for correcting!